fsw/cfe-core/src/inc/cfe_error.h

```diff
@@ -808,12 +808,12 @@
 #define CFE_OS_ERR_INVALID_PRIORITY     (OS_ERR_INVALID_PRIORITY) /**< @copydoc OS_ERR_INVALID_PRIORITY */
 #define CFE_OS_ERROR_TASK_ID            (OS_ERROR_TASK_ID) /**< @brief This doesn't actually exist */
 #define CFE_OS_SEM_UNAVAILABLE          (OS_SEM_UNAVAILABLE) /**< @brief This doesn't actually exist */
-#define CFE_OS_FS_ERROR                 (OS_FS_ERROR) /**< @copydoc OS_FS_ERROR */
-#define CFE_OS_FS_ERR_INVALID_POINTER   (OS_FS_ERR_INVALID_POINTER) /**< @copydoc OS_FS_ERR_INVALID_POINTER */
+#define CFE_OS_ERROR                 (OS_ERROR) /**< @copydoc OS_ERROR */
```

Comment:
Leave as CFE_OS_FS_ERROR, this whole group is deprecated in #582 but need to deprecate before removing.  Same for CFE_OS_FS_ERR_INVALID_POINTER.